index.html

```diff
@@ -840,14 +836,10 @@ <h2>DID Parameters</h2>
 based on the <code>query</code> component (See <a href="#query"></a>).
         </p>
         <p>
-Some DID parameter names (for example, for service selection) are
-completely independent of any specific <a>DID method</a> and MUST always
-function the same way for all <a>DIDs</a>. Other DID parameter names (for
-example, for versioning) MAY be supported by certain <a>DID methods</a>, but
-MUST operate uniformly across those <a>DID methods</a> that do support them.
-        </p>
-        <p>
-The following table defines a set of DID parameter names.
+Some DID parameters are supported by all <a>DID methods</a> and others need
```

Comment:
I think there is a subtle difference between saying that "DID parameters are supported by all DID methods" and "DID parameters are independent of DID methods". For example, the `service` DID parameter has nothing to do with DID methods, it affects a later stage of the DID URL dereferencing process. Do you maybe have an idea how to preserve the intention of the original language?